di-ipv-evcs-stub/lambdas/src/services/evcsService.ts

```diff
@@ -0,0 +1,97 @@
+import { DynamoDB, GetItemInput, PutItemInput } from "@aws-sdk/client-dynamodb";
+import { marshall, unmarshall } from "@aws-sdk/util-dynamodb";
+
+import PostRequest from "../domain/postRequest";
+import VcStateMetadata from "../domain/vcStateMetadata";
+import ServiceResponse from "../domain/serviceResponse";
+import AllState from "../domain/enums/allState";
+import EvcsVcItem from "../model/evcsVcItem";
+
+import { config } from "../common/config";
+import { getSsmParameter } from "../common/ssmParameter";
+import { v4 as uuid } from "uuid";
+
+const dynamoClient = config.isLocalDev
+  ? new DynamoDB({
+      endpoint: config.localDynamoDbEndpoint,
+      region: config.region,
+    })
+  : new DynamoDB({ region: config.region });
+
+export async function processPostUserVCsRequest(
+    userId: string,
+    postRequest: PostRequest
+  ): Promise<ServiceResponse> {
+  console.info(`Post user record.`);
```

Comment:
Is this useful? It should probably either include some useful information, or be removed